multichat/chat/consumers.py

```diff
@@ -1,6 +1,8 @@
 import json
 from channels import Channel
 from channels.auth import channel_session_user_from_http, channel_session_user
+
+from .settings import MSG_TYPE_LEAVE, MSG_TYPE_ENTER, NOTIFY_USERS_ON_ENTER_OR_LEAVE_ROOMS
```

Comment:
Hi there!

First of all, I thnik this is a very good package. Good work :). Now, an answer to this comment.

AFAIK the use of an inner file in a Django package (e.g. `package/settings.py`) is a common way to define default values for configuration parameters that are optional.

In case of general use configuration variables (used in more than one place), using the same `getattr(var, module, default)` sentence in several sites seems to violate the DRY principle so I think this change may be a a good way to proceed. If you think it isn't, please let me know why.

Cheers!